code/modules/paperwork/pen.dm

```diff
@@ -332,3 +332,15 @@
 			P.contact_poison_poisoner = user.name
 			add_attack_logs(user, P, "Poison pen'ed")
 			to_chat(user, "<span class='warning'>You apply the poison to [P].</span>")
+
+// CHAMELEON PEN
+/obj/item/pen/chameleon
+	name = "chameleon pen"
+	desc = "It's a surprisingly boring-looking pen! Try using it to leave a signature."
+	var/forge_name = null
+
+/obj/item/pen/chameleon/attack_self(mob/living/user)
+	forge_name = tgui_input_text(user, "Enter the name of the person whose signature you want to forge", "Forge name", max_length = MAX_NAME_LEN)
+
+/obj/item/pen/chameleon/on_write(obj/item/paper/P, mob/user)
+	P.forge_name_forger = user.name
```

Comment:
forger_name_forgers is being used?

Reply:
I saw that poison pen is doing same for poisoner. It's my understanding that the admin would be able to look and see who made a fake sign with this var. Other than this it doesn't have any purpose.

Reply:
Do you think its important for the pen to have it? I dont think the stamp keeps track of it and would be of same importance. I would like to know what others think.

Reply:
I asked a few admins i know, it would be better to have it in. Maybe there is a better way to make it, like using add_attack_log()? This system is not perfect, but i prefer admins have a little bit more info in case they need it.